airflow/models/__init__.py

```diff
@@ -39,4 +38,5 @@
 
 # Load SQLAlchemy models during package initialization
 # Must be loaded after loading DAG model.
+# noinspection PyUnresolvedReferences
```

Comment:
I like green checkmark when I solved all the warnings in PyCharm. I think it's also an indication that we deliberately ignored some problem which otherwise could be overlooked. It helps me (and I think I know at least few more people using pyCharm/IntelliJ and actually paying attention to warnings. I usually try to leave any file I am checking "clean" with regards to PyCharm's default warning settings. But If it's really annoying I can remove it.